components/mpas-seaice/cime_config/buildnml

```diff
@@ -805,8 +805,6 @@ def buildnml(case, caseroot, compname):
             lines.append('    <var name="iceAgeTendencyTransport"/>')
             lines.append('    <var name="iceAgeCell"/>')
```

Comment:
It appears that the age tracers `iceAgeCell` and `firstYearIceAreaCell` are no longer in the output file, but the `iceAgeTendency` terms are still present.

Reply:
You have highlighted a deficiency in MPAS-SeaIce in that the fields were empty, but the headers remained.  I have removed all iceAge output from buildnml. New ncdump -h is:

<img width="1751" alt="Screen Shot 2023-12-02 at 4 25 05 PM" src="https://github.com/E3SM-Project/E3SM/assets/14379090/40f798a1-65e5-4ab9-af93-7a2cdabc268d">